docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,271 @@
+
+* The browser support data for this feature, which consists of the version number for each of the browsers that support the feature.
+
+#### Understand where browser support data comes from
+
+The browser support data for your feature is what determines its baseline status. For example, if your feature is supported on Chrome 66, Chrome Android 66, Edge 79, Firefox 76, Firefox Android 79, Safari 12.1, and Safari iOS 12.2, then it's baseline high, because those versions have been released long enough ago.
+
+The browser support data that we use comes from the [browser-compat-data project](https://github.com/mdn/browser-compat-data/) (BCD), which you need to understand before continuing to set the status of your feature.
+
+BCD contains browser support data for individual constituent of web features, such as CSS properties, JavaScript statements, HTML elements and attributes, DOM interfaces, methods, and events.
+For example, BCD contains the list of browsers, and their versions, that support the [`grid-template-rows` CSS property](https://github.com/mdn/browser-compat-data/blob/main/css/properties/grid-template-rows.json), which is a constituent of the [grid](https://github.com/web-platform-dx/web-features/blob/main/features/grid.yml) feature.
+
+We refer to each entry that contains browser support data in BCD as a _BCD key_. In the `grid-template-rows` example, the BCD key is `css.properties.grid-template-rows`.
+
+Features in the web-features project are associated to one or more BCD keys. For example, the grid feature is associated to multiple BCD keys, which together, describe the overall grid feature: `css.properties.display.grid`, `css.properties.display.inline-grid`, `css.properties.grid`, `css.properties.grid-area`, and more.
+
+#### Associate BCD keys with your feature
+
+To set the status of your new feature, you must associate your feature with one or more BCD keys. There are two cases for associating your feature to BCD keys:
+
+* Either BCD already defines references to your new feature. In this case, you don't need to explicitly list the BCD keys that your feature depends on.
+
+  For example, the [grid](https://github.com/web-platform-dx/web-features/blob/main/features/grid.yml) feature doesn't list any BCD keys, because the BCD project already maps the right BCD keys to the `grid` web-features ID. See the [`web-features:grid` search results](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data%20%22web-features%3Agrid%22&type=code) in BCD.
+
+  Here is the grid feature YAML file content, showing that no BCD keys appear in the file:
+
+  ```yaml
+  name: Grid
+  description: CSS Grid is a two-dimensional layout system, which lays content out in rows and columns.
+  spec: https://drafts.csswg.org/css-grid-3/
+  group: grid
+  caniuse: css-grid
+  ```
+
+  You should always check first if BCD doesn't already map to your feature ID by searching for `web-features:<your-feature-id>` in the BCD source code.
```

Comment:
Instead of searching BCD, one can just run `npm run dist` and see if that results in some computed status.